modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
+. In the *+Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
+. Configure the pipeline using either the *Pipeline builder* view or the *YAML view*.
++
+[NOTE]
+====
+The *Pipeline builder* view supports a limited number of fields whereas the *YAML view* supports all available fields. Optionally, you can also use the Operator-installed, reusable snippets and samples to create detailed Pipelines.
+====
++
+.YAML view
+image::op-pipeline-yaml.png[]
++
+To configure your pipeline using the *Pipeline Builder*:
+
+.. Enter a unique name for the pipeline.
+.. Select a task from the *Select Task* list to add a task to the pipeline. This example uses the *s2i-nodejs* task.
+*** To add sequential tasks to the pipeline, click the plus icon to the right or left of the task, and from the *Select Task* list, select the task you want to add to the pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* task to add an *openshift-client* task.
+*** To add a parallel task to the existing task, click the plus icon displayed next to the task, and from the *Select Task* list, select the parallel task you want to add to the pipeline.
 +
 .Pipeline Builder
 image::op-pipeline-builder.png[]
 +
-. Click *Add Resources* to specify the name and type of resources that the Pipeline Run will use. These resources are then used by the Tasks in the Pipeline as inputs and outputs.
+.. Click *Add Resources* to specify the name and type of resources that the pipeline run will use. These resources are then used by the tasks in the pipeline as inputs and outputs.
 For this example:
-.. Add an input resource. In the *Name* field, enter `Source`, and from the *Resource Type* drop-down list, select *Git*.
-.. Add an output resource. In the *Name* field, enter `Img`, and from the *Resource Type* drop-down list, select *Image*.
-. The *Parameters* for a Task are prepopulated based on the specifications of the Task. If required, use the *Add Parameters* link to add additional parameters.
+... Add an input resource. In the *Name* field, enter `Source`, and then from the *Resource Type* drop-down list, select *Git*.
+... Add an output resource. In the *Name* field, enter `Img`, and then from the *Resource Type* drop-down list, select *Image*.
+.. Optional: The *Parameters* for a task are prepopulated based on the specifications of the task. If required, use the *Add Parameters* link to add additional parameters.
 +
-. A *Missing Resources* warning is displayed on a Task if the resources for the Task are not specified. Click the *s2i-nodejs* Task to see the side panel with details for the Task.
+.. A *Missing Resources* warning is displayed on a task if the resources for the task are not specified. Click the *s2i-nodejs* task to see the side panel with details for the task.
```

Comment:
Should this start with _If..._ instead so it is a step similar to the others?

Reply:
This part is not under my current user story although adding an "If" here won't make much of a difference because the tool actually sometimes shows warning and sometimes doesn't.